manager/integration/tests/common.py

```diff
@@ -4261,6 +4261,20 @@ def create_backing_image_with_matching_url(client, name, url):
     return bi
 
 
+def check_backing_image_disk_map_status(client, bi_name, expect_cnt, diskmap_status): # NOQA
```

Comment:
1. Rename input `diskmap_status` -> `expect_disk_state`
2. This function means that the state of all disk files of a backing image should be the same. And the count should match. Or you just want to check if there are **enough** disk files being in the expected state, regardless of some other disk files being in an arbitrary state.

Reply:
HI @shuo-wu ,
I changed the variable name to `expect_disk_state` and modified function `check_backing_image_disk_map_status`, let the function check counts of specific disk map status were equal to we expected or not. thank you.